src/main/java/com/crud/tasks/repository/TaskRepository.java

```diff
@@ -0,0 +1,19 @@
+package com.crud.tasks.repository;
+
+import com.crud.tasks.com.crud.task.domain.Task;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public interface TaskRepository extends CrudRepository<Task,Long> {
+
+    @Override
+    List<Task> findAll();
+
+    List<Task> findById();
```

Comment:
to powinno byc 
`Optional<Task> findById(Long id);` id jest:
- parametrem
- jest unikalny wiec optional (albo jest albo nie)